docs/docs/featurizer/hashing.md

```diff
@@ -0,0 +1,62 @@
+This page discusses some properties of the `HashingFeaturizer`.
+Note that this featurizer is a *sparse* featurizer.
+
+The featurizer is a wrapper a around scikit-learn's
+[HashingVectorizer](https://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.text.HashingVectorizer.html).
+It uses the "hashing trick" to transform input texts to a sparse vector by mapping each token to a column index using a
+fixed hash function. The featurizer has no state and cannot be trained. For a small number of columns (defined by the
+`n_features` parameter), hash coalitions are more likely, meaning that two words can get mapped to the
+same index.
```

Comment:
I might add that it's recommended to add more than one hashingvectorizer but with a different `n_features` value. The idea is that if you have hashing happening with bucketsizes 1999, 2000, 2001 that the odds of all three having a collision is very small.